<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeScreen</class>
 <widget class="QWidget" name="HomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="header">
     <property name="font">
      <font>
       <family>Corbel</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>DaSSCo Mass Digitizer App - Version 1.0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="separatorLine">
     <property name="frameShape">
      <enum>QFrame::Shape::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblSelectInstitution">
     <property name="text">
      <string>Please choose your institution to proceed:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="lstSelectInstitution">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblUsername">
     <property name="text">
      <string>Specify username:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inpUsername"/>
   </item>
   <item>
    <widget class="QLabel" name="lblPassword">
     <property name="text">
      <string>Specify password:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inpPassword">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblSelectCollection">
     <property name="text">
      <string>Choose a collection to log in:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="lstSelectCollection">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblIncomplete">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string>color: red;</string>
     </property>
     <property name="text">
      <string>Please fill in username/password!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblAuthError">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string>color: red;</string>
     </property>
     <property name="text">
      <string>Authentication Error!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblCollError">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string>color: red;</string>
     </property>
     <property name="text">
      <string>Please choose a collection!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnExit">
     <property name="locale">
      <locale language="English" country="Denmark"/>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
